src/components/HandlerManager/HandlerManager.js

```diff
@@ -20,7 +20,15 @@ class HandlerManager extends React.Component {
   constructor(props) {
     super(props);
     const { event, stripes, modules, data } = props;
-    this.components = getEventHandlers(event, stripes, modules.handler, data);
+    const uniqueModules = Object.values(modules)
```

Comment:
@NikitaSedyx I think the original idea for this was to only support the `handler` modules. Could you explain why was this changed here?

Reply:
because non-handler modules can't handle events like `LOGIN`, according to https://github.com/folio-org/stripes/blob/master/doc/dev-guide.md#handlers-and-events 
```
Stripes modules of any type can provide a handler; the handler module type is provided for the case where the handler is the only thing the module provides, i.e. there is no main-screen app and no settings.
```
and it's strange that we provide a handler in app or settings module, but it can't handle events

Reply:
as an alternative, we can add `handler` type to our module, but it will contradict with description of handler module, additionally it will require to refactor ProfileDropdown logic to handle duplicates (actually the same logic that is applied here)

IMO following current documentation,  HandlerManager should support handler of all module types, not just handlers

Reply:
I agree with @mkuklis here. I don't think this needs to change; you just need to add the `handler` to the `stripes.actsAs` array in the `package.json` of the ui-$whatever repo. Have you tried that?

Reply:
yes, in this case we have duplicates in ProfileDropdown actions because it lists action for each value in `actsAs` - as I mentioned above it's not a problem as we just need to add module uniqueness 

but here I have another point, for action menu we can add config to any module, define handler to return component and it will work without `stripes.actsAs: handler`, at the same time to handle events via `HandlerManager` we need to define module as a handler, it's usage inconsistency

Reply:
so we have to use cases to use handlers one doesn't require module as a handler, another one requires

Reply:
@zburke @mkuklis reverted changes for HandlerManager, probably handler docs section should be updated, at least 2 devs were confused by it :)

Reply:
Thank you @NikitaSedyx. I agree with you that the docs should include the information about how to turn the existing ui-module into a `handler` and that the `HandlerManager` is intended to be used only by `hander` modules. Do you think that would be sufficient?

Reply:
I believe it should be enough, I was a bit confused by 2 points
1. any module can provide handler (but in fact ignored by `HandlerManager`)
2. handler type is used for case when module provides only handler

looks like what you mentioned covers them

Reply:
Thank you @NikitaSedyx